server/src/utils/file.ts

```diff
@@ -58,7 +58,7 @@ export const sendFile = async (
 
     res.header('Content-Type', file.contentType);
     if (file.fileName) {
-      res.header('Content-Disposition', `inline; filename="${file.fileName}"`);
+      res.header('Content-Disposition', `inline; filename*=UTF-8''${encodeURI(file.fileName)}`);
```

Comment:
Filenames containing a comma still fail in chrome, but this fixes it:

```suggestion
      res.header('Content-Disposition', `inline; filename*=UTF-8''${encodeURIComponent(file.fileName)}`);
```

Reply:
Nice catch!